subprojects/studio-plugin/build.gradle.kts

```diff
@@ -19,6 +19,7 @@ java {
 intellij {
     pluginName.set("gradle-profiler-studio-plugin")
     version.set("2021.1.1")
+    type.set("IC")
```

Comment:
What does this do? (Might be worth documenting the intention in a comment.)

Reply:
Removed, default is "IC" anyway. Basically it sets "Ide type" to use. IC is Intellij community that Android Studio is based on: https://github.com/JetBrains/gradle-intellij-plugin#intellij-platform-properties